packages/@okta/i18n/src/properties/login_ht.properties

```diff
@@ -738,8 +737,8 @@ account.unlock.unlocked.title = Kont lan Te Debloke San Pwobl\u00E8m
 account.unlock.unlocked.desc = Ou kapab konekte pandan w ap itilize non itilizat\u00E8 ak modpas ou deja genyen an.
 account.unlock.code.notReceived = \u00C8ske w pa te resevwa yon k\u00F2d Debloke gras ak im\u00E8l
 account.unlock.noFactorsEnabled = Pa genyen okenn opsyon deblokaj ki egziste. Tanpri kontakte administrat\u00E8 ou an.
-account.unlock.authenticatorRequired.single = Pou debloke kont ou an, seleksyone otantifikat\u00E8 sa a.
-account.unlock.authenticatorRequired.multiple = Pou debloke kont ou an, seleksyone youn nan otantifikat\u00E8 sa yo.
+account.unlock.authenticatorRequired.single = Pou debloke kont ou an, seleksyone Authenticator sa a.
+account.unlock.authenticatorRequired.multiple = Pou debloke kont ou an, seleksyone youn nan Authenticator sa yo.
```

Comment:
@lesterchoi-okta Does this look correct given the long thread about capitalization of "Authenticator"?